internal/dockerfile2dot/load.go

```diff
@@ -11,11 +11,11 @@ import (
 
 // LoadAndParseDockerfile looks for the Dockerfile and returns a
 // SimplifiedDockerfile.
-func LoadAndParseDockerfile(inputFS afero.Fs) (SimplifiedDockerfile, error) {
-	content, err := afero.ReadFile(inputFS, "Dockerfile")
+func LoadAndParseDockerfile(inputFS afero.Fs, filename string) (SimplifiedDockerfile, error) {
+	content, err := afero.ReadFile(inputFS, filename)
 	if err != nil {
 		if errors.Is(err, fs.ErrNotExist) {
-			return SimplifiedDockerfile{}, errors.New("could not find any Dockerfile in the current working directory")
+			return SimplifiedDockerfile{}, errors.New("could not find a Dockerfile at " + filename)
```

Comment:
I changed the error message here to include the filename; is the wording OK?

Reply:
Wording is perfect, thanks! 🎉

To make the error message even more clear, we could add a `./` before the filename. That way, instead of 

`Error: could not find a Dockerfile at Dockerfile1`

it would say

`Error: could not find a Dockerfile at ./Dockerfile1`

In a future version, we could resolve the file path to its canonical absolute value — but that's really out of scope for this PR 😄

Reply:
Giving it a bit more thought, we should really go with https://pkg.go.dev/path/filepath#Abs instead – my suggestion above is pretty ugly 😄